tools/update-ecma-version-tests.js

```diff
@@ -42,7 +43,7 @@ function findTests(directory) {
  * @returns {void}
  */
 function outputResult(result, testResultFilename) {
-    `export default ${tester.getAstCode(result)};`.to(testResultFilename);
+    fs.writeFileSync(testResultFilename, `export default ${tester.getAstCode(result)};`);
```

Comment:
Due to an update to `shelljs`, the previous code was no longer usable.

https://github.com/eslint/espree/pull/600